azure/folder.go

```diff
@@ -65,11 +68,14 @@ func ConfigureFolder(prefix string, settings map[string]string) (storage.Folder,
 			return nil, NewCredentialError(AccessKeySetting)
 		}
 	}
+	if environmentName, ok = settings[EnvironmentName]; !ok {
+		environmentName = "AzurePublicCloud"
```

Comment:
I would advise extracting this to a const at the top of the file and naming it like default environment

Reply:
Thank you for the advice. I created new const and set the default value "AzurePublicCloud" in my most recent [commit](https://github.com/wal-g/storages/pull/43/commits/ed5d7c1f42f0093181f1d3c30f0fe0c7849b937a)